pkgs/applications/video/obs-studio/default.nix

```diff
@@ -53,6 +53,14 @@ in mkDerivation rec {
     fetchSubmodules = true;
   };
 
+  patches = [
+    # Lets obs-browser build against CEF 90.1.0+
```

Comment:
can we upstream these patches and add a link to them here?

Reply:
I tried, they weren't very receptive. Maintainers seem to be of the "we don't accept changes unless you're using specifically an officially supported version of our dependencies" type which is fair enough from a maintenance perspective (especially given that they're probably dealing with gamers most of the time lol) but not one I really wish to support in nixpkgs given that we're dealing with a browser engine here. See https://github.com/obsproject/obs-browser/issues/296 and https://github.com/obsproject/obs-browser/pull/297

Reply:
(marking as resolved as I don't expect any luck convincing them otherwise)